api/schema.py

```diff
@@ -2,21 +2,22 @@
 from marshmallow import fields, validate
 
 
+class AddUserSchema(ma.Schema):
+    id = fields.String()
+    username = fields.String(validate=validate.Length(min=5, max=50), required=True)
+    email = fields.String(validate=validate.Length(min=10, max=60), required=True)
+    password = fields.String(validate=validate.Length(min=4, max=60), required=True)
+    photo = fields.String()
+
+
 class UserSchema(ma.Schema):
     id = fields.String()
     username = fields.String(validate=validate.Length(min=5, max=50), required=True)
     email = fields.String(validate=validate.Length(min=10, max=300), required=True)
     profile_image = fields.String(
         validate=validate.Length(min=1, max=10), required=True
     )
```

Comment:
these seem the same to me
whats the difference?
since a DB  model needs only a single  schema